### ISSUE TITLE
Turbo Everdrive: where to find Nantettatte Engine?

### ISSUE THREAD

**Issue by frederic-mahe** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#21]
> v4.1 – Game Series Collections update by tjstogy!
> - Added Nantettatte Engine (PD) v1 Aetherbyte – Included with permission, thanks Arkhan!

The ROM available on [Aetherbyte](http://www.aetherbyte.com/aetherbyte-nantettatte_engine.html) has a hash value different from the one listed in the Turbo SMDB:
```
# SMDB file
e3380f3b592a85ebd187caf290dcb5a0d7154970c08365cd0f92e97607d7273a    Turbo EverDrive/4 Homebrew/Nantettatte Engine (PD) v1 Aetherbyte.pce
#  Aetherbyte download
783a049de279c56f102aef71a64d85883e9014da244754f9e288bc85fbe43147  Nantettatte Engine.pce
```
Where can one find the correct ROM?

**Comment by SmokeMonsterPacks** [comment]
Oh, cool. I bet they updated it. I'll get that in the next TED update. Thanks!

**Comment by SmokeMonsterPacks** [comment]
Yep, I just tested and the new revision has a sweet, sweet intro. Very cool. I'll get that added to the next update. Thanks!

**Comment by frederic-mahe** [comment]
Nantettatte Engine (PD) v2 is now included (commit dc9022b0daafc508c9e7d013fe6f046942c7928e).
